### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by psnider** [DanH91/ttd-angular2#2]
- node v4.4.7
- npm 2.15.8

npm install fails with this output:

```
npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/Cellar/node4-lts/4.4.7/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.4.7
npm ERR! npm  v2.15.8
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package reflect-metadata@0.1.3 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer angular2@2.0.0-beta.17 wants reflect-metadata@0.1.2
```

**Comment by danh91** [comment]
Thanks for reporting this issue. Pull request are welcome

**Comment by JordanAlimi** [comment]
This has been fixed in #3 . The issue was versioning, thanks for the feedback.

This project needs Angular 2 alpha, if you want to use a higher version you'd need to wait until we update the structure.

**Comment by danh91** [comment]
Thanks @JordanAlimi for the fix. The issue will be closed
